src/index.css

```
@@ -416,45 +416,148 @@
+     We do NOT use @tailwindcss/typography (prose) here because
+     our surface is dark glass with CSS-variable-driven accent
+     colors — prose's opinionated color assumptions would need
+     so many overrides that hand-rolling is cleaner.
      ============================================================ */
 
-  .chat-markdown { @apply whitespace-pre-wrap break-words; }
+  .chat-markdown {
+    @apply leading-relaxed break-words;
+    /* Prevent pre/code from blowing out narrow containers */
+    overflow-wrap: break-word;
+    word-break: break-word;
```

Stylelint (17.3.0)

[error] Unexpected deprecated keyword "break-word" for property "word-break" (declaration-property-value-keyword-no-deprecated)

(declaration-property-value-keyword-no-deprecated)